example/lambda.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-const awsServerlessExpress = require('aws-serverless-express')
+const awsServerlessExpress = require(process.env.NODE_ENV === 'test' ? '../index' : 'aws-serverless-express')
```

Comment:
Use the library version instead of the installed version from NPM during tests. This is required for:

1. Writing tests for new features
1. Coverage report